stacks/HDP/2.0.6/services/CDAP/metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<!--
+   Copyright © 2016 Cask Data, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+   use this file except in compliance with the License. You may obtain a copy of
+   the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+   License for the specific language governing permissions and limitations under
+   the License.
+-->
+<metainfo>
+  <schemaVersion>2.0</schemaVersion>
+  <services>
+    <service>
+      <name>CDAP</name>
+      <extends>common-services/CDAP/3.5.0-SNAPSHOT</extends>
```

Comment:
I could do that, but as it is, currently, I can checkout this repo and symlink stuff into the right places, then `git pull` to update. Helpful for testing... especially as I am working on multiple features at once and the "SNAPSHOT" style versioning doesn't really lend itself to knowing which package contains which code... it's all just a timestamp.